styles/desktop.css

```diff
@@ -7,22 +7,21 @@
     margin: 16px 0 16px 143px;
   }
 
-  @keyframes rotation {
-    from {
-      transform: rotate(0deg);
-    }
+  .icon-menu {
+    display: none;
+  }
 
-    to {
-      transform: rotate(360deg);
-    }
+  .click-event {
+    display: inline-block;
   }
 
-  .icon-menu {
+  .close-btn {
     display: none;
   }
 
   .menu-lists {
     display: flex;
+    flex-direction: row;
     list-style-type: none;
```

Comment:
- [ ] Also make sure the popup layer fit the entire height of the screen even when the user ted to scroll all the way down. they don't need to see the content of the page.

![image](https://user-images.githubusercontent.com/97100091/210395740-ffe92df3-3b13-495a-903b-d0365827ca22.png)